.github/workflows/integration-test.yml

```diff
@@ -76,6 +76,8 @@ jobs:
           tutor local do dump-data-to-clickhouse --options "--object course_overviews"
           make extract_translations
           tutor local do import-assets
+          tutor local do collect-dbt-lineage
+          tutor local run lms python manage.py lms transform_tracking_logs --source_provider LOCAL --source_config '{"key": "/openedx/data/", "prefix": "tracking.log", "container": "logs"}' --destination_provider LRS --transformer_type xapi
```

Comment:
This is a somewhat random addition to this PR, but will test that we don't break the backfill command again